content/en/docs/getting-started/setup/create-cluster/google.md

```diff
@@ -7,8 +7,261 @@ publishdate: 2017-02-01
+
+Run Jenkins X Boot at the command-line `jx` and include the
+pre-configured `jx-requirements-gke.yml` file:
+
+    jx boot -r jx-requirements-gke.yml
+
+The interactive command-line tool guides you through the process of
+configuring and installing the resources needed to complete your GitOps
+development environment:
+
+1.  The program prompts you to choose your Google Cloud Project from the
+    available list
+
+2.  The program prompts you for the Cluster Name, which you previously
+    chosen (in this example, `acmecluster1`).
+
+3.  The program prompts you for a `Git Owner` or your GitHub account (in
+    this example, `acmecorp`).
+
+4.  The program prompts you to enter a Storage Bucket URLs for saving
+    various storage resources:
+
+    1.  Logs: You can use an existing storage bucket URL or press
+        **Enter** and a bucket will be created for you.
+
+    2.  Reports: You can use an existing storage bucket URL or press
+        **Enter** and a bucket will be created for you.
+
+    3.  Repository: You can use an existing storage bucket URL or press
+        **Enter** and a bucket will be created for you.
+
+5.  If you are prompted to upgrade your Jenkins X binaries or Platform
+    at this stage of installation, press **N** to refuse the
+    upgrade.
+
+6.  The program prompts you to enter a Jenkins X administrator Username.
+    You can use your current GitHub account for consistency (in this
+    example, `acmecorp`).
+
+    1.  The program prompts you to enter a password for the Jenkins X
+        administrator
+
+7.  The program asks you for a Pipeline Bot Git username (for example
+    `acmebot`). This creates a bot for automating your GitOps process by
+    promoting code from your development environment to your staging
+    and/or production environments.
+
+    1.  Create a new GitHub account strictly for the build control bot
+
+    2.  Invite the bot account to your organization
+
+    3.  Create a Git token for the Pipeline Bot with the correct permissions via
+        this [GitHub
+        Link](https://github.com/settings/tokens/new?scopes=repo,read:user,read:org,user:email,write:repo_hook,delete_repo)
+        and **copy the 40 character token**. 
+        
+    > **Warning**: The 40 character token generated by GitHub is only
+    >shown once so you must copy this immediately before you close the
+    >browser tab or window, as the token cannot be retrieved once it is
+    >displayed.
+
+    1.  Paste the copied GitHub token created for the Pipeline Bot when
+        prompted
+
+8.  The program prompts you to provide a hash-based message
+    authentication code (`HMAC`) token for authenticating incoming
+    webhooks from your Git provider and accept PR data and other
+    payloads to your cluster. Press **Enter** to create a token or
+    copy and paste one into the command line if you have an existing
+    token.
+
+9.  The program asks if you would like to configure an external Docker
```

Comment:
Explain how to configure the product to use other container registries or provide a link to the page with more info.